gnc/navigator_controller/src/mrac_controller.py

```diff
@@ -336,7 +377,12 @@ def increment_reference(self):
 
         # Use model drag to find drag force on virtual boat
         twist_body = R_ref.T.dot(np.concatenate((self.v_ref, [self.w_ref])))
-        D_body = np.array([p if v > 0 else n for p, n, v in zip(self.D_body_positive, self.D_body_negative, twist_body)])
+        D_body = np.zeros_like(twist_body)
```

Comment:
zeros_like() is the same as zero(len())
That line initializes D_body, it does not do anything to twist_body except reference its len().